src/features/user/Settings/EditProfile/AddressManagment/AddressManagement.module.scss

```diff
@@ -1,8 +1,29 @@
 @import '../../../../../theme/theme';
 
-.addressManagement {
+$border-radius-sm: 8px;
+$border-radius-lg: 16px;
+$padding-standard: 24px;
```

Comment:
Personally, I feel these variables didn't need to be abstracted, and the code is harder to read with them.

Do we want to change padding-standard in one place and affect both SingleAddress and the address shown in the type update modals?